### ISSUE TITLE
TravisCI reports progress with one dot per line

### ISSUE THREAD

**Issue by mlocati** [sebastianbergmann/phpunit#2388]
I noticed that recently PHPUnit started reporting progress in TravisCI with one dot per line.
It's not an important issue, but that makes reading the log a bit harder.

For instance:
- before: https://travis-ci.org/concrete5/concrete5/jobs/178392806
- after: https://travis-ci.org/concrete5/concrete5/jobs/181618862

Is this a TravisCI issue, a PHPUnit issue or an issue with the integration of the two?

**Comment by mlocati** [comment]
Cross-reference: https://github.com/travis-ci/travis-ci/issues/6979

**Comment by sebastianbergmann** [comment]
I cannot reproduce this, see https://travis-ci.org/sebastianbergmann/phpunit/jobs/181264097 for instance.

Are you sure that you did not change anything other than PHPUnit?

**Comment by mlocati** [comment]
- Progress shown correctly:  
  date: 2016-11-30T17:26:16Z  
  commit: b0fa09b67d37bc47b6fa801779fde9a634bd18ae  
  https://travis-ci.org/concrete5/concrete5/jobs/180152674
- Progress reported with one dot per line:  
  date: 2016-12-01T17:14:12Z  
  commit: ea3a119922e48ec631c34dcca4a67d5126a2bda1  
  https://travis-ci.org/concrete5/concrete5/jobs/180461325

Changes between the commits:
https://github.com/concrete5/concrete5/compare/b0fa09b67d37bc47b6fa801779fde9a634bd18ae...ea3a119922e48ec631c34dcca4a67d5126a2bda1

As you can see, the only change has nothing to do with phpunit or travis

**Comment by sebastianbergmann** [comment]
Thank you for your report.

Please provide a **minimal**, **self-contained**, reproducing test case that shows the problem you are reporting.

Without such a minimal, self-contained, reproducing test case I will not be able to investigate this issue.

**Comment by mlocati** [comment]
I wish I had the possibility to give some more detail, but I don't really know how could I do that.

The PHP version is unchanged, the PHPUnit version is unchanged, the changes to our code are not relevant, I simply don't know what could lead to this problem, except some changes in the Travis environment.

That's why I'm going to close this issue here, and hopefully solve this problem in the [TravisCI](https://github.com/travis-ci/travis-ci/issues/6979) bug report.

(I just hoped someone else already saw this problem)

**Comment by sebastianbergmann** [comment]
How are you invoking PHPUnit? Could it be related to https://twitter.com/mmoreram/status/806141340866244609?

**Comment by mlocati** [comment]
Yes, maybe: we call phpunit via composer: see [composer.json](https://github.com/concrete5/concrete5/blob/develop/composer.json#L80) and [.travis.jml](https://github.com/concrete5/concrete5/blob/develop/.travis.yml#L30), but we've been doing that [since October](https://github.com/concrete5/concrete5/commit/ac89a572051d9df0ed1467e6d42489d3af16d200), but tests was running fine since a few days ago (eg https://travis-ci.org/concrete5/concrete5/jobs/180152674 ), so I'm not 100% sure...

BTW, looking at the TravisCI log, we can see that composer version changed from version 1.2.2 to 1.2.3, so that may be the cause.

I'll open another PR on composer issue tracker :wink:

**Comment by mlocati** [comment]
Others have this problem with composer: see https://github.com/composer/composer/issues/5936 and https://github.com/composer/composer/issues/5933

Thanks for help!
